contrib/inventory/foreman.py

```diff
@@ -117,10 +124,27 @@ def read_settings(self):
         self.cache_path_inventory = cache_path + "/%s.index" % script
         self.cache_path_params = cache_path + "/%s.params" % script
         self.cache_path_facts = cache_path + "/%s.facts" % script
+        self.cache_path_hostcollections = cache_path + "/%s.hostcollections" % script
         try:
             self.cache_max_age = config.getint('cache', 'max_age')
         except (ConfigParser.NoOptionError, ConfigParser.NoSectionError):
             self.cache_max_age = 60
+
+        # If the FOREMAN_ORGANIZATION environment variable is set to the
+        # label of a Satellite organization, only hosts from that
```

Comment:
With commit 944f712, I've changed the meaning of the FOREMAN_ORGANIZATION variable. It now is the organization name (not the label). I've also added an example in this comment.

Already replaced Satellite with Foreman before.